### ISSUE TITLE
wrong config directory

### ISSUE THREAD

**Issue by zbence** [bbinet/docker-salt-master#3]
The readme says I should use /config in the container for the config files but it's not working. The config dir is /etc/salt

**Comment by bbinet** [comment]
You are right, I've changed the default config dir without updating the doc.
